doc/api.md

```diff
@@ -588,6 +592,10 @@ expression. In all cases the following data is available to all expressions:
 Additionally, for attribute expressions (`expression`, `entity_name`), `entityNameExp` and metadata expressions
 (`expression`) measures are available in the **context** used to evaluate them.
 
+Attribute metadata and Static Attribute metadata are available in the **context** under the following convention:
+`metadata.AttributeName.MetadataName` or `metadata.StaticAttributeName.MetadataName` in a similar way of defined for
+[Context Broker](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#metadata-support)
```

Comment:
It seems this paragraph is the only new thing in documentation, all the rest seems to be re-formatting.

@AlvaroVega could you confirm, pls?

Reply:
```suggestion
Attribute metadata and Static Attribute metadata are available in the **context** under the following convention:
`metadata.<AttributeName>.<MetadataName>` or `metadata.<StaticAttributeName>.<MetadataName>` in a similar way of defined for
[Context Broker](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#metadata-support)
```

Using the `<...>` syntax would make more explicit that they refer to actual names, different to `metadata` which is a literal token.

Reply:
Yes, it is